paymentsheet/src/test/java/com/stripe/android/paymentsheet/state/DefaultPaymentElementLoaderTest.kt

```diff
@@ -1351,6 +1352,59 @@ internal class DefaultPaymentElementLoaderTest {
             .isEqualTo(CardBrandChoiceEligibility.Eligible(listOf()))
     }
 
+    @Test
+    fun `Returns correct Link signup mode if has used Link before`() = runTest {
```

Comment:
This adds two tests that were missing to fully cover the Link signup mode scenarios.